locales-src/asrouter.ftl

```diff
@@ -81,3 +81,26 @@ cfr-whatsnew-button =
   .tooltiptext = What’s New
 
 cfr-whatsnew-panel-header = What’s New
+
+## Recommended features
+
+cfr-feature-type-recommended = Recommended Feature
+
+## Bookmark Sync
+
+cfr-sync-bookmarks-header = Get this bookmark on your phone
+cfr-sync-bookmarks-body = Take your bookmarks, passwords, history and more everywhere you’re signed into { -brand-product-name }.
+cfr-sync-bookmarks-cta-button = Turn on { -sync-brand-short-name }
```

Comment:
Looks like CFR buttons need accesskey attributes

Reply:
should the reference just be branding/brand.ftl? Or something else? It shows up with /official in the gecko-strings: 
https://hg.mozilla.org/l10n/gecko-strings/file/default/browser/branding/official/brand.ftl

Reply:
I'll add .accesskey :)

Reply:
https://github.com/mozilla/activity-stream/pull/5231#discussion_r311748899

Reply:
Shoot sorry, I didn't refresh and didn't have that comment 👍